src/core.jl

```diff
@@ -208,92 +92,61 @@ function trainGigaSOM(som::Som, train::DataFrame;
 
     train = convertTrainingData(train)
 
-    # set default radius:
+    dTrain = distribute(train) #this slices the data
+    distribute_darray(:__trainGigaSOM, dTrain) #this actually sends them to workers
+    res = trainGigaSOM(som, :__trainGigaSOM, dTrain.pids,
+                       kernelFun=kernelFun,
+                       metric=metric,
+                       knnTreeFun=knnTreeFun,
+                       rStart=rStart, rFinal=rFinal, radiusFun=radiusFun,
+                       epochs=epochs)
+    undistribute_darray(:__trainGigaSOM, dTrain)
+    return res
+end
+
+function trainGigaSOM(som::Som, dataVal, workers::Array{Int64};
+                      kernelFun::Function = gaussianKernel,
+                      metric = Euclidean(),
+                      knnTreeFun = BruteTree,
+                      rStart = 0.0, rFinal=0.1, radiusFun=linearRadius,
+                      epochs = 10)
 
+    # set the default radius
     if rStart == 0.0
         rStart = √(som.xdim^2 + som.ydim^2) / 2
         @info "The radius has been determined automatically."
     end
 
+    # get the SOM neighborhood distances
     dm = distMatrix(som.grid, som.toroidal)
 
     codes = som.codes
 
-    nWorkers = nprocs()
-    dTrain = distribute(train)
-
     for j in 1:epochs
+        @info "Epoch $j..."
```

Comment:
Maybe here could also be a `printLevel` so that the entire output could be muted at once (similar to the i/o splitting).

Reply:
Is there some standard method for that or should we make one? (In C I'm usually just dumping stuff to `stderr` and everyone is welcome to ignore it with a redirect :D)

Reply:
EDIT: there's apparently @debug and standardized loglevels; I'll use that.

Reply:
cool - sure, why not. maybe let's do another PR just for that